NFT/src/pages/api/metadata.ts

```diff
@@ -0,0 +1,16 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import pinataSDK from "@pinata/sdk";
+
+const pinata = new pinataSDK({ pinataJWTKey: process.env.PINATA_JWT });
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+    if (req.method === "PUT") {
+        const { ipfsHash, metadata } = JSON.parse(req.body);
+
+        const response = await pinata.hashMetadata(ipfsHash, metadata);
+        return res.send(response);
+    }
```

Comment:
We should put this in a try catch block, also validating for null cases for `ipfsHash` and `metadata`.